cardverificationsheet/src/main/java/com/stripe/android/cardverificationsheet/cardverifyui/CardVerifyActivity.kt

```diff
@@ -75,7 +76,7 @@ data class RequiredCardDetails(
 private val MINIMUM_RESOLUTION = Size(1067, 600) // minimum size of OCR
 
 @Keep
-open class CardVerifyActivity : SimpleScanActivity<RequiredCardDetails>() {
+open class CardVerifyActivity : SimpleScanActivity<RequiredCardDetails?>() {
```

Comment:
Merchants have historically asked to create their own UIs. This is why the UI is set up programmatically rather than through XML configuration, so that merchants are able to build their own extending our base.